src/go/rucio-dataset-mon-go/main.go

```diff
@@ -5,33 +5,49 @@ import (
 	"github.com/dmwm/CMSMonitoring/src/go/rucio-dataset-mon-go/controllers"
 	"github.com/dmwm/CMSMonitoring/src/go/rucio-dataset-mon-go/routes"
 	"github.com/gin-gonic/gin"
+	"golang.org/x/sync/errgroup"
 	"log"
-	"os"
+	"net/http"
+	"time"
 )
 
-// main
-func main() {
-	// Check GO_ENVS_SECRET_PATH is set
-	envVal, present := os.LookupEnv("GO_ENVS_SECRET_PATH")
-	if present == true {
-		log.Fatal("Please set $GO_ENVS_SECRET_PATH environment variable. Exiting...")
-	} else {
-		log.Printf("GO_ENVS_SECRET_PATH is : %t\n", envVal)
-	}
+// Reference: https://github.com/gin-gonic/gin/issues/346
+
+var (
+	g errgroup.Group
+)
+
```

Comment:
Please add into main option to provide server version and info. see https://github.com/dmwm/dbs2go/blob/master/main.go#L21, then add makefile which will setup `gitVersion`, see https://github.com/dmwm/dbs2go/blob/master/Makefile#L2, and add new end-point which will return server info, see https://github.com/dmwm/dbs2go/blob/master/web/handlers.go#L284 Such that someone can query your server and get the info, e.g.
```
scurl https://cmsweb.cern.ch/dbs/prod/phys03/DBSReader/serverinfo
[{"dbs_version":"v00.05.32","server":"dbs2go git=v00.05.32 go=go1.18.3 date=2022-13-06"}]
```
As you can see it is very useful to know from the server itself which version it is running